I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 12, in script
    init python:
  File "game/script.rpy", line 20, in <module>
    peachVisited
NameError: name 'peachVisited' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 12, in script
    init python:
  File "G:\Downloadings\renpy-6.99.14.3-sdk\renpy\ast.py", line 862, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "G:\Downloadings\renpy-6.99.14.3-sdk\renpy\python.py", line 1888, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 20, in <module>
    peachVisited
NameError: name 'peachVisited' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.14.3.3347
RenPy 1.0
Wed May 30 05:22:32 2018
